style.css

```diff
@@ -272,3 +318,91 @@ h4 {
 .s-title {
   padding-right: 180px;
 }
+
+/* Contact Section */
+
+.contact {
+  width: 375px;
+  border-top-left-radius: 90px;
+  padding: 150px 24px 50px 24px;
+  background: #6070ff;
+  display: flex;
+  flex-direction: column;
+  margin: 80px 0 20px 0;
+  justify-content: center;
+  align-content: center;
+  background-image: url(./images/contact.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+/* .contact.container {
+  margin-bottom: 50px;
+} */
+
+.contact-text {
+  margin: 12px 0;
+}
+
+.contact-title h2 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+}
+
+.contact p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+}
+
+.contact-form {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] DRY(Do not Repeat Yourself) is a requirement, but since the other reviewers overlooked it, I'm stating it as optional, It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and then assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.